WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -184,6 +184,10 @@
 
         <%= f.input :guests_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, currency_selector: "#competition_currency_code" %>
 
+        <div class="guest-no-entry-fee-options">
+          <%= f.input :free_entry_text, as: :radio_buttons, collection: [ :none, :anyone, :restricted ] %>
```

Comment:
Maybe (I'm not sure myself, you will have to try) you can re-use the enum that I suggested above and pass it to the `collection` parameter here. (Or perhaps a `collection` parameter isn't even necessary anymore when Ruby detects that the `:free_entry_text` field is backed by an enum)

Reply:
Yep it worked, luckily sombody installed `enum_helper` before me